### ISSUE TITLE
Document PyPI installation options

### ISSUE THREAD

**Issue by lukaszachy** [teemtee/tmt#2465]
One can customize what to install from PyPI ('tmt[all]', 'tmt[provision-virtual]',..._) but we don't have it documented.
The only list of options is in the pyproject.toml itself.

Not sure where to document this though. The 'Readme.rst' is common for github and PyPI overview and the https://tmt.readthedocs.io/en/stable/overview.html#install is too long and quite rpm focused.

**Comment by psss** [comment]
Yeah, makes sense to have this properly documented. What about the overview/install section?

* https://tmt.readthedocs.io/en/stable/overview.html#install

@lukaszachy, would that be sufficient?
